float32 motor_response
float32 reward
float32 qvalue
uint32[3] state
uint32 explored_states